Neo 0.6.1: error in reading .abf file header (same as #302)

Hello,

I'm sorry, but I think I'm having the same issue as a user in a previous thread. I keep getting the following error:

File "/Users/matt/anaconda3/lib/python3.6/site-packages/neo/rawio/axonrawio.py", line 458, in parse_axon_soup
'This file does not contain clampex, axoscope or clampfit in the header'

AssertionError: This file does not contain clampex, axoscope or clampfit in the header

I checked the solution previously stated, however it appears in the latest version I downloaded on Monday (3.26.2018) that that had been fixed, e.g. in the latest version it says:
goodstart = -1
for key in [b'AXENGN', b'clampex', b'Clampex',
b'CLAMPEX', b'axoscope', b'Clampfit']:
# goodstart = big_string.lower().find(key)
goodstart = big_string.find(key)
if goodstart != -1:
break
assert goodstart != -1, 
'This file does not contain clampex, axoscope or clampfit in the header'
big_string = big_string[goodstart:]
strings = big_string.split(b'\x00')

Here is the .abf in the hopes it can be looked at.
https://drive.google.com/open?id=1CAXLYRNwLsg5vXIH8SABKod1TCrvwJK0

Thank you,
Matthew Clark

--- a/neo/rawio/axonrawio.py
+++ b/neo/rawio/axonrawio.py
@@ -449,7 +449,7 @@ def parse_axon_soup(filename):
             big_string = f.read(sections['StringsSection']['uBytes'])
             goodstart = -1
             for key in [b'AXENGN', b'clampex', b'Clampex',
-                        b'CLAMPEX', b'axoscope', b'Clampfit']:
+                        b'CLAMPEX', b'axoscope', b'AxoScope', b'Clampfit']:
                 # goodstart = big_string.lower().find(key)
                 goodstart = big_string.find(key)
                 if goodstart != -1:

